### DISCUSSION TITLE
反応しないキーについて

### DISCUSSION THREAD

**Discussion by hiro1115** [Yowkees/keyball#316] [Q&A]
自作キーボードど素人です。
この度keyball 61を購入し制作しています。

![image](https://github.com/Yowkees/keyball/assets/142844754/765d7f31-d557-4179-b84b-0338964bb2b2)
赤枠の4つのキーを表面からピンセットでショートさせても反応がありません。ダイオードなど新品に付け替えなどしたのですが改善ありません。
どのようにしたら改善するでしょうか。

また、ショートさせて反応があった部位でもキースイッチを取り付けたところほとんどのキーの反応がなくなりました。
キーキャップの足折れは無かったのでキーソケットのハンダが悪いことが原因でしょうか？
失敗だらけの回路をお見せいたします。改善すべき点をご教授いただけますと幸いです。
![image](https://github.com/Yowkees/keyball/assets/142844754/73481fb4-e211-4cac-b050-4d73cb2879e1)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

ほとんどのキーが反応しませんか！？
おそらく多くがキーソケットのテンプラ（半田付け不良）と思われます。
キースイッチはつけたままで、ボトムプレートを外し、裏側から１つずつ半田付け確認してください。
写真を見るとはんだの量が多いようですので、もう少し落として、下記リンクの写真のように横から覗いて端子の下に半田が流れ込んでいることをご確認ください。
https://github.com/Yowkees/keyball/blob/main/keyball61/doc/rev1/images/kb44_080.jpg

以上、お手数ですがご対応お願いいたします。

**Reply by hiro1115** [reply]
ありがとうございます。
やり直してみます！
